pkgs/tools/misc/datefmt/default.nix

```
@@ -0,0 +1,22 @@
+{ lib, stdenv, fetchurl }:
+
+stdenv.mkDerivation rec {
+  pname = "datefmt";
+  version = "0.2.1";
+
+  src = fetchurl {
+    url = "http://cdn.jb55.com/tarballs/${pname}/${pname}-${version}.tar.gz";
```

```suggestion
    url = "http://cdn.jb55.com/tarballs/datefmt/datefmt-${version}.tar.gz";
```
There's really no need to substitute the name everywhere, makes it only harder to read imo